perf/lua/benchmark.lua

```diff
@@ -0,0 +1,63 @@
+local json = require('json')
+
+local M = {}
+
+local function format_report(bench)
+    local output_format = bench.output_format
+    local results = bench.results
+    local report = ''
+    if output_format == 'json' then
+        -- The output should have the same format as the Google
+        -- Benchmark.
```

Comment:
Google Benchmark supports three different output formats: `console`, `json`, and `csv` [^1]. I would say explicitly that Google Benchmark JSON output format. Probably link to Google Benchmark and a sample with output will be useful in comments.

[^1]: https://github.com/google/benchmark/blob/main/docs/user_guide.md

Reply:
Clarified the comment and added the link, thanks!